torch_geometric/distributed/local_graph_store.py

```diff
@@ -18,10 +18,39 @@ def __init__(self):
         self._edge_attr: Dict[Tuple, EdgeAttr] = {}
         self._edge_id: Dict[Tuple, Tensor] = {}
 
+        self.num_partitions: int = 1
+        self.partition_idx: int = 0
+        # node_partition_book:  mapping between node ids and partition idx
+        self.node_pb: Union[Tensor, Dict[NodeType, Tensor]] = None
+        # edge_partition_book:  mapping between edge ids and partition idx
+        self.edge_pb: Union[Tensor, Dict[EdgeType, Tensor]] = None
+        # meta information related to partition and graph store info
+        self.meta: Optional[Dict[Any, Any]] = None
+        # partition labels
+        self.labels: Union[Tensor, Dict[EdgeType, Tensor]] = None
```

Comment:
self.meta is the partition information like num_partition, is_hetero, edge_type, .. which is easy/ very helpful for later code simplicity and you don't need judge based on dict or not,etc.  self.labels is the whole dataset information for labels y and also easy for later code simplicity otherwise you need put one more argument from top to low level code function.